File: python practice/20260611_Hello World PA.py
import sys

# 讀取所有標準輸入，並自動依換行或空白分割成陣列
inputs = sys.stdin.read().split()

# 遍歷所有輸入的數字，每有一個數字就印出一次 Hello World
for _ in inputs:
    print("Hello World")

File: C Language/20260611_Hello World PA.c
#include <stdio.h>

int main() {
    int n;
    
    // 持續讀取輸入，直到檔案結束 (EOF) 變更為止
    while (scanf("%d", &n) != EOF) {
        printf("Hello World\n");
    }
    
    return 0;
}